src/openrct2/world/Map.cpp

```diff
@@ -63,8 +63,14 @@ using namespace OpenRCT2;
 /**
  * Replaces 0x00993CCC, 0x00993CCE
  */
-const CoordsXY CoordsDirectionDelta[] = { { -32, 0 },   { 0, +32 },   { +32, 0 },   { 0, -32 },
-                                          { -32, +32 }, { +32, +32 }, { +32, -32 }, { -32, -32 } };
+const std::array<CoordsXY, 8> CoordsDirectionDelta = { CoordsXY{ -COORDS_XY_STEP, 0 },
+                                                       CoordsXY{ 0, +COORDS_XY_STEP },
```

Comment:
Can you add a line break after `CoordsDirectionDelta = {` and indent the contents _one_ step (i.e. 4 spaces)? Makes for a more legible block.

Reply:
Done, but that required turning off clang-format for that piece of code.